roles/ci_gen_kustomize_values/molecule/default/converge.yml

```diff
@@ -25,8 +25,20 @@
         (ansible_user_dir,
          'src/github.com/openstack-k8s-operators/architecture') | path_join
       }}
+    artifacts: "{{ cifmw_ci_gen_kustomize_values_basedir }}/artifacts"
   tasks:
-    # We first create a dedicated keypair.
+    - name: Ensure architecture repository is building
+      ansible.builtin.shell:
+        chdir: "{{ cifmw_ci_gen_kustomize_values_architecure_repo }}/{{ item.path }}"
+        cmd: >-
+          {{ ansible_user_dir }}/bin/kustomize build >
+          {{ artifacts}}/vanilla-{{ item.name }}.yaml
```

Comment:
a space missing after `artifacts`

Reply:
more than that, I should use the `artifacts` parameter instead of re-writing the whole path in some other places. Updated.